testdata/project-v2-addon/Makefile

```diff
@@ -75,9 +75,11 @@ deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
 	$(KUSTOMIZE) build config/default | kubectl apply -f -
 
-undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/config.
-	$(KUSTOMIZE) build config/default | kubectl delete -f -
 
+KB_UNDEPLOY_IGNORE_NOT_FOUND ?= 0
+undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/config.
+	$(KUSTOMIZE) build config/default | kubectl delete --ignore-not-found=$(KB_UNDEPLOY_IGNORE_NOT_FOUND) -f -
```

Comment:
What do you think about instead create an ENV VAR here we do something like:
```
action:
    echo argument is $(argument)
You would then call it make action argument=something
```
Also, would be nice add the info that args can be informed at the end in the comment `## Undeploy controller from the K8s cluster specified in ~/.kube/config.`. Why? That is the description of the target when we run make --help. 

Also, it would allow people to pass any arg instead of an specific one and would remove the need to use and doc env vars.

Reply:
/ok-to-test

Reply:
I'm still not sure how to deal with defaults in this case, though. What do you think about something like:

```Makefile
ifndef ignore-not-found
  ignore-not-found = no
endif

undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
    $(KUSTOMIZE) build config/default | kubectl delete --ignore-not-found=$(ignore-not-found) -f -
```

Reply:
@camilamacedo86 I've updated the code with my suggestion and also spread it to the _uninstall_ target. Seemed to make sense.